### PR TITLE
generating test data content

### PR DESCRIPTION
**Pull request by lauramthomas**
### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation fix (typos, incorrect content, missing content, etc.)

### Checklist

- [ ] I have read the [contributing](https://github.com/saucelabs/sauce-docs/blob/main/docs/contributing.md) document.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: docs/api-testing/composer.md [hunk 1]
```diff
@@ -369,6 +369,150 @@ To convert a timestamp from a payload response to milliseconds:
...
+* `F.lastName()` - Generates a last name
+* `F.emailAddress()` - Generates an email address
+* `F.password(<minimumLength,maximumLength,includeUppercase,includeSpecial,includeDigit>)` - Generates a password
+* `F.creditCardNumber()` - Generates a credit card number
+* `F.creditCardExpiry()` - Generates a credit card expiration date
+* `F.integer(<min,max>)` - Generates an integer
+
+For the full list of methods, see [Test Data Methods](#test-data-methods).
+
+These methods can be used anywhere you can write a variable, such as inside a SET (variable) or in any part of the request (body, header, param, etc.).
```
Comment (lysannep) [inline docs/api-testing/composer.md:386]:
```suggestion
These methods can be used anywhere you can write a variable, such as inside a Set (variable) or in any part of the request (body, header, param, etc.).
```

File: docs/api-testing/composer.md [hunk 2]
```diff
@@ -369,6 +369,150 @@ To convert a timestamp from a payload response to milliseconds:
 
 For more information, see [Expression Language Extensions](/api-testing/composer/logical-components/#expression-language-extensions).
 
+## Generating Test Data
+If your API or test requires random names, emails, or different types of input data, you can generate those directly in Sauce Labs API Testing. You can directly reference the method in your variable, API call, or anywhere in the test where you can enter the `${F.<methodName()>}` syntax.
+
+* `F.fullName()` - Generates a full name
```
Comment (lysannep) [inline docs/api-testing/composer.md:375]: Should the method descriptions end with a period?

**Review by lysannep — DISMISSED** [review]

**Review by spider-sauce — APPROVED** [review]
LGTM from a file perspective.

File: docs/api-testing/composer.md [hunk 3]
```diff
@@ -369,6 +369,150 @@ To convert a timestamp from a payload response to milliseconds:
...
+These examples are of the most common places where you may need to generate data, but these methods can be added anywhere you can use a variable.
+
+### Test Data Methods
+
+#### Addresses and Countries
+* `F.streetName()` - Generates a street name
+* `F.streetAddressNumber()` - Generates an address number
+* `F.streetAddress()` - Generates a street and address number. If secondary is specified, this method provides an apartment number.
+* `F.secondaryAddress()` - Generates an apartment number
+* `F.zipCode()` - Generates a ZIP code. If a state has been provided, a proper ZIP code for the state is provided. Valid only for US states.
```
Comment (LoryApiFortress) [inline docs/api-testing/composer.md:460]: @lauramthomas remove this statement :"If a state has been provided, a proper ZIP code for the state is provided"

**Review by lysannep — APPROVED** [review]

**Pull request merged**